stories/documentation/structure/cards/cards-action.stories.ts

```diff
@@ -0,0 +1,40 @@
+import { Meta, Story } from '@storybook/angular';
+
+interface CardsActionStory {
+	elevated: boolean;
+}
+
+export default {
+	title: 'Documentation/Structure/Cards/Action',
+	elevated: {
+		control: {
+			type: 'boolean',
+		}
+	},
+} as Meta;
+
+function getTemplate(args: CardsActionStory): string {
+	const elevated = args.elevated ? `mod-elevated` : '';
+	return `
+	<div class="card mod-action ${elevated}">
+		<div class="card-content">
+			<h2 class="card-title">Titre de la carte</h2>
+			<p>Contenu de la carte</p>
+		</div>
+		<div class="card-action">
+			<button type="button" class="actionIcon" luTooltip="Modifier">
+			  <span aria-hidden="true" class="lucca-icon icon-arrowFullEast"></span>
+			  <span class="u-mask">Accéder</span>
+			</button>
+		</div>
+	</div>
+	`
```

Comment:
Design will probably ask for the entire card to be clickable. A frquent use case we have is that these cards act as links to content. I think we will need this component to handle the `<a class="card mod-action">` use case.

Reply:
After a chat with @vvalentin-lucca it seems that having a whole card as link element can lead to accessibility issue. 
It seems that the best way is to let Angular handle the click : https://codepen.io/matuzo/full/ZEqbXOM
I'll add a `mod-action `anyway to cover hover effect.